eslint.config.js

```diff
@@ -0,0 +1,28 @@
+import js from '@eslint/js'
+import globals from 'globals'
+import reactHooks from 'eslint-plugin-react-hooks'
+import reactRefresh from 'eslint-plugin-react-refresh'
+import tseslint from 'typescript-eslint'
```

Comment:
**issue**: The import statement for 'typescript-eslint' seems incorrect. It should be '@typescript-eslint/eslint-plugin' and '@typescript-eslint/parser' for proper ESLint integration with TypeScript.